elijah/main.js

```diff
@@ -0,0 +1,38 @@
+// Your card and deque factory code should be loaded into the browser before this file
+// by the main.html container file.
+// At this point, both makeCard and makeDeque should be defined.
+
+
+// 2b:
+// make a deque instance to store a full deck of cards:
+var deckOfCards = makeDeque(makeCard.fullSet);
+
+
+deckOfCards.sort(suitOrder);
+deckOfCards.cut();
+assert(deckOfCards.top().name() === 'King of Diamonds', 'Failed King of Diamonds test');
+//
+deckOfCards.sort(alphOrder);
+assert(deckOfCards.bottom().name() === 'Ace of Clubs', 'Failed Ace of Clubs test');
+assert(deckOfCards.top().name() === 'Two of Spades', 'Failed Two of Spades test');
+
+// 2c:
+// make a deque instance to store student names:
+
```

Comment:
2c looks good. Code is working, with the slight exception that currently your sort is placing your names in reverse order. deckOfNames.top() produces Julius right now, where it should produce Jaka.